invesalius/navigation/markers.py

```diff
@@ -117,6 +117,16 @@ def DeleteMultiple(self, marker_ids):
 
         self.SaveState()
 
+    def AddMultiple(self, markers_list):
+        Publisher.sendMessage("Set markers list rendering", render=False)
+        for marker in markers_list:
+            self.AddMarker(marker, render=False)
+
+        Publisher.sendMessage("Set markers list rendering", render=True)
+        Publisher.sendMessage("Render volume viewer")
```

Comment:
The rendering volume should only happen if the navigation is off. Either you could check the navigation status or disable the load marker button when navigation is on. I think the second option is better.

Reply:
I think the first option is better here, since AddMultiple is general functionality that could be used from more places later.